composer.json

```diff
@@ -14,11 +14,13 @@
         "laravel/framework": "5.5.*",
         "laravel/tinker": "~1.0",
         "league/flysystem-aws-s3-v3": "^1.0",
-        "predis/predis": "^1.1"
+        "predis/predis": "^1.1",
+        "swagger-api/swagger-ui": "^3.2"
     },
     "require-dev": {
         "filp/whoops": "~2.0",
         "fzaninotto/faker": "~1.4",
+        "laravel/homestead": "^6.2",
```

Comment:
do we really need to add this dependency?

Reply:
Can be removed. Is only needed, if you use homestead per project approach.